liota/dccs/iotcc.py

```diff
@@ -157,7 +159,15 @@ def on_response(msg, reg_resp_q):
                     self._store_reg_entity_attributes("Devices", entity_obj.name, self.reg_entity_id,
                                                       entity_obj.entity_type, None)
 
-            return RegisteredEntity(entity_obj, self, self.reg_entity_id)
+            _reg_entity_obj = RegisteredEntity(entity_obj, self, self.reg_entity_id)
+            if self.sys_properties is not None and self.sys_properties:
+                if ast.literal_eval(self.sys_properties):
```

Comment:
What if this returns a String or a number of a tuple? I think we need a check if it is an instance of dictionary.